libs/geoservices/api/src/lib/geoservices-api.component.spec.ts

```diff
@@ -8,6 +14,7 @@ describe('GeoservicesApiComponent', () => {
 
   beforeEach(async(() => {
     TestBed.configureTestingModule({
+      imports: [CommonModule, UILayoutModule, UIScrollToModule, RouterTestingModule, BrowserAnimationsModule],
```

Comment:
From what I've been able to dig up, `CommonModule` isn't required for tests? Tests pass without it.

Clean orphan import statement.